### ISSUE TITLE
FairseqMultimodel doesn't have forward_decoder

### ISSUE THREAD

**Issue by twelveth** [pytorch/fairseq#1315]
When I try to inference model, inheriting from `FairseqMultimodel` (I've created custom `MultilingualLSTMModel` with `MultilingualTransformerModel` as example) with either fairseq-generate or fairseq-interactive, it crashes, saying
```
AttributeError: 'MultilingualLSTMModel' object has no attribute 'forward_decoder'
```
There is no such method both in `FairseqMultiModel` and `BaseFairseqModel` (which is parent of `FairseqMultiModel`).

I suppose, the same issue will be with multilingual transformer, because both it and my class inherit from `FairseqMultiModel`.

**Comment by trilobi** [comment]
+1

**Comment by lematt1991** [comment]
Resolved by 4c5934ac61354d9b6d164f7317905e4ac2ae1064
